### PR TITLE
Added links for adding EditorConfig files in IDEs

### LINKED ISSUE #30959: Add links to how to add EditorConfig files in IDEs
Working with a developer new to .NET this page didn't help direct them to 'how to add EditorConfig' in the various tools.

Please add links to 'how' to do this:
- VSMac: https://docs.microsoft.com/en-us/visualstudio/mac/editorconfig
- VS: https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2022#add-and-remove-editorconfig-files

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e8c8357d-11a6-6f12-b044-5ff44df65d10
* Version Independent ID: 0c103fd7-5cc4-2726-40f8-d759419ae635
* Content: [.NET code style rule options - .NET](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/code-style-rule-options)
* Content Source: [docs/fundamentals/code-analysis/code-style-rule-options.md](https://github.com/dotnet/docs/blob/main/docs/fundamentals/code-analysis/code-style-rule-options.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

### PR DESCRIPTION
**Pull request by ankitwww**
## Summary

Fixes #30959. Added links for how to add EditorConfig files in different IDEs.

### CODE REVIEW

File: docs/fundamentals/code-analysis/code-style-rule-options.md [hunk 1]
```diff
@@ -39,7 +39,12 @@ To help you get started, here's an example *.editorconfig* file with the default
 > For more information, see [Option format](style-rules/language-rules.md#option-format).
 
 > [!TIP]
-> In Visual Studio, you can add the following default .NET .editorconfig file to your project from the **Add New Item** dialog box.
+>
+> - In Visual Studio, you can add the following default .NET .editorconfig file to your project from the **Add New Item** dialog box. Refer [this article](/visualstudio/ide/create-portable-custom-editor-options?#add-and-remove-editorconfig-files) for detailed steps.
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/code-style-rule-options.md:43]:
```suggestion
> - In Visual Studio, you can add the following default .NET .editorconfig file to your project from the **Add New Item** dialog box. For detailed steps, see [Add and remove EditorConfig files](/visualstudio/ide/create-portable-custom-editor-options?#add-and-remove-editorconfig-files).
```

File: docs/fundamentals/code-analysis/code-style-rule-options.md [hunk 2]
```diff
@@ -39,7 +39,12 @@ To help you get started, here's an example *.editorconfig* file with the default
...
+>
+> - For Visual Studio for Mac, Refer [this](/visualstudio/mac/editorconfig) article.
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/code-style-rule-options.md:45]:
```suggestion
> - For information about using .editorconfig files in Visual Studio for Mac, see [Creating and editing a custom EditorConfig file](/visualstudio/mac/editorconfig).
```

File: docs/fundamentals/code-analysis/code-style-rule-options.md [hunk 3]
```diff
@@ -39,7 +39,12 @@ To help you get started, here's an example *.editorconfig* file with the default
 > For more information, see [Option format](style-rules/language-rules.md#option-format).
 
 > [!TIP]
-> In Visual Studio, you can add the following default .NET .editorconfig file to your project from the **Add New Item** dialog box.
+>
+> - In Visual Studio, you can add the following default .NET .editorconfig file to your project from the **Add New Item** dialog box. Refer [this article](/visualstudio/ide/create-portable-custom-editor-options?#add-and-remove-editorconfig-files) for detailed steps.
+>
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/code-style-rule-options.md:44]:
```suggestion
```

File: docs/fundamentals/code-analysis/code-style-rule-options.md [hunk 4]
```diff
@@ -39,7 +39,12 @@ To help you get started, here's an example *.editorconfig* file with the default
...
+> - For Visual Studio for Mac, Refer [this](/visualstudio/mac/editorconfig) article.
+>
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/code-style-rule-options.md:46]:
```suggestion
```

File: docs/fundamentals/code-analysis/code-style-rule-options.md [hunk 5]
```diff
@@ -39,7 +39,12 @@ To help you get started, here's an example *.editorconfig* file with the default
...
+> - For Visual Studio Code, You can [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) plugin.
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/code-style-rule-options.md:47]:
```suggestion
> - In Visual Studio Code, you can use the [EditorConfig for VS Code plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) to override user/workspace settings with settings found in .editorconfig files.
```

**Review by gewarren — COMMENTED** [review]
Thanks for adding these links. I added a little more detail in my suggestions.

File: docs/fundamentals/code-analysis/code-style-rule-options.md (hunk 2 above)
Comment (ankitwww) [reply docs/fundamentals/code-analysis/code-style-rule-options.md]: Thanks for the suggestion!

File: docs/fundamentals/code-analysis/code-style-rule-options.md [hunk 6]
```diff
@@ -39,7 +39,10 @@ To help you get started, here's an example *.editorconfig* file with the default
 > For more information, see [Option format](style-rules/language-rules.md#option-format).
 
 > [!TIP]
-> In Visual Studio, you can add the following default .NET .editorconfig file to your project from the **Add New Item** dialog box.
+>
+> - In Visual Studio, you can add the following default .NET .editorconfig file to your project from the **Add New Item** dialog box. For detailed steps, see [Add and remove EditorConfig files](/visualstudio/ide/create-portable-custom-editor-options?#add-and-remove-editorconfig-files).
+> - For information about using .editorconfig files in Visual Studio for Mac, see [Creating and editing a custom EditorConfig file](/visualstudio/mac/editorconfig).
+> - For Visual Studio Code, You can use the [EditorConfig for VS Code plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) to override user/workspace settings with settings found in .editorconfig files.
```
Comment (gewarren) [inline docs/fundamentals/code-analysis/code-style-rule-options.md:45]:
```suggestion
> - For Visual Studio Code, you can use the [EditorConfig for VS Code plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) to override user/workspace settings with settings found in .editorconfig files.
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**